lib/components/Menu.js

```diff
@@ -133,11 +133,18 @@ var Menu = module.exports = React.createClass({
     return options;
   },
 
+  buildName: function() {
+    var classes = 'Menu';
```

Comment:
:+1: yeah usually the build process would inline this so it's not loading anything extra.